app/__init__.py

```diff
@@ -68,4 +68,13 @@ def create_app(config_name):
     from bulk import bulk as bulk_blueprint
     app.register_blueprint(bulk_blueprint, url_prefix='/bulk')
 
+    from descriptor import descriptor as descriptor_blueprint
+    app.register_blueprint(descriptor_blueprint, url_prefix='/descriptor')
+
+    from resource import resource as resource_blueprint
```

Comment:
If I'm understanding correctly, the `resource` blueprint is for making changes to single resources, while the `bulk` blueprint is for making changes to many resources at once. Seems like they should have similar names? Perhaps they should be called `individual_resource` and `bulk_resource`? Or maybe they actually aren't that similar? What do you think?